src/ImageSharp/Dithering/ErrorDiffusion/IErrorDiffuser.cs

```diff
@@ -0,0 +1,34 @@
+﻿// <copyright file="IErrorDiffuser.cs" company="James Jackson-South">
+// Copyright (c) James Jackson-South and contributors.
+// Licensed under the Apache License, Version 2.0.
+// </copyright>
+
+namespace ImageSharp.Dithering
+{
+    using System;
+
+    /// <summary>
+    /// Encapsulates properties and methods required to perfom diffused error dithering on an image.
+    /// </summary>
+    public interface IErrorDiffuser
+    {
+        /// <summary>
+        /// Gets the dithering matrix
+        /// </summary>
+        byte[][] Matrix { get; }
```

Comment:
Ehh not getting answers on discussions if I don't keep pressing F5. 
It's too late here to write working code :( I can do it next time. Will sketch BulkOperations instead, keep your eyes on gitter ;)